sensor_test.py

```diff
@@ -51,9 +54,7 @@ def test_reboot(get_sensor_info, reboot_sensor):
 
     log.info("Reboot sensor")
     reboot_response = reboot_sensor()
-    assert (
-        reboot_response == "rebooting"
-    ), "Sensor did not return proper text in response to reboot request"
+    assert reboot_response["result"] == "rebooting", "Sensor did not return proper text in response to reboot request"
```

Comment:
In order to avoid doing
```
assert reboot_response["result"]
```
and instead do (as before)
```
assert reboot_response
```
you could apply the same changes you made in `get_sensor_info()` to other functions that call sensor methods.